pdr_backend/lake/test/conftest.py

```diff
@@ -601,16 +601,16 @@ def setup_data(
     )
 
     gql_tables = {
-        "pdr_predictions": Table(Prediction, ppss),
-        "pdr_truevals": Table(Trueval, ppss),
-        "pdr_payouts": Table(Payout, ppss),
-        "pdr_slots": Table(Slot, ppss),
+        "pdr_predictions": NamedTable.from_dataclass(Prediction),
+        "pdr_truevals": NamedTable.from_dataclass(Trueval),
+        "pdr_payouts": NamedTable.from_dataclass(Payout),
+        "pdr_slots": NamedTable.from_dataclass(Slot),
     }
 
-    gql_tables["pdr_predictions"].append_to_storage(preds)
-    gql_tables["pdr_truevals"].append_to_storage(truevals)
-    gql_tables["pdr_payouts"].append_to_storage(payouts)
-    gql_tables["pdr_slots"].append_to_storage(slots)
+    gql_tables["pdr_predictions"].append_to_storage(preds, ppss)
+    gql_tables["pdr_truevals"].append_to_storage(truevals, ppss)
+    gql_tables["pdr_payouts"].append_to_storage(payouts, ppss)
+    gql_tables["pdr_slots"].append_to_storage(slots, ppss)
```

Comment:
I'm seeing ppss being passed around so much.
It's growing too.
And it's a configuration file.

Isn't there a better way?